.azure/pipelines/ci.yml

```diff
@@ -520,6 +520,7 @@ stages:
         $(_PublishArgs)
         $(_InternalRuntimeDownloadArgs)
       installNodeJs: false
+      disableComponentGovernance: true
       skipComponentGovernanceDetection: true
```

Comment:
do we still need `skipComponentGovernanceDetection` and the `GC_RAN` build step in default-build.yml❔

Reply:
I adopted a "if it ain't broke don't fix it mentality here". It appears to me that if `disableComponentGovernance` and `skipComponentGovernanceDetection` are set to the same value, then the same behavior will persist.